python/nano/test/tf/keras/test_fit.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import os
+
+os.system("pip install ray[default]")
+os.system("pip install -U intel_tensorflow")
+
+import pathlib
```

Comment:
You generally should not add logic in the global scope. You should wrap them in the test function.